packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -597,19 +595,21 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     if (this.props.disabled && (ev.which === KeyCodes.enter || ev.which === KeyCodes.space)) {
```

Comment:
Consider reworking the check/else if checks so that they all don't have to check this.props.disabled

Reply:
Alternatively this could be written like `if (this.props.disabled) { if(enter || space){cancelEvent} return;} <rest of the code/>` then everything wouldn't be nested in an if/else